src/proposals/api/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from proposals.api.views import ProposalAPIView
+
+urlpatterns = [
+    path("", ProposalAPIView.as_view()),
```

Comment:
讀者看到 `/api/proposals` 會覺得這隻 api 可能會直接給出 proposal data，覺得可以改成更好被解讀的 path，for example
```suggestion
    path("summary/", ProposalAPIView.as_view()),
```